docs/rules/order-in-components.md

```diff
@@ -123,3 +123,28 @@ export default Component.extend({
   }
 });
 ```
+
+#### Custom Prop Ordering
```

Comment:
It feels too verbose to have an entire section dedicated to explaining this. Instead, I would recommend simply adding a single sentence in the existing `## Configuration` section.

```
If you would like to keep a specific property in a particular order, then you can use the `custom:myPropertyName` syntax in the order list to specify where `myPropertyName` should go.
```